exercise-stringer.go

```diff
@@ -1,27 +1,25 @@
 package main
 
-import ("fmt"
-	"strings"
+import (
+	"fmt"
 )
 type IPAddr [4]byte
 
 // TODO: Add a "String() string" method to IPAddr.
 func (ip IPAddr) String() string {
-	IPAddrString := []string{}
-	//lặp các phần tử lấy value
-	for _, value := range ip {
-		IPAddrString = append(IPAddrString, fmt.Sprint(int(value))) //mỗi lần lặp thì thêm 1 phần tử vào IPAddrString
+	var str string
```

Comment:
k cần dùng for, có thể dùng đơn giản thế này thôi
return fmt.Sprintf("%d.%d.%d.%d", ip[0], ip[1], ip[2], ip[3])